staging/src/k8s.io/sample-controller/README.md

```diff
@@ -31,6 +31,46 @@ The sample controller uses [client-go library](https://github.com/kubernetes/cli
 The details of interaction points of the sample controller with various mechanisms from this library are
 explained [here](docs/controller-client-go.md).
 
+## Fetch sample-controller and its dependencies
+
+Like the rest of Kubernetes, sample-controller has used
+[godep](https://github.com/tools/godep) and `$GOPATH` for years and is
+now adopting go 1.11 modules.  There are thus two alternative ways to
+go about fetching this demo and its dependencies.
+
+### Fetch with godep
+
+When NOT using go 1.11 modules, you can use the following commands.
+
+```sh
+go get -d k8s.io/sample-controller
+cd $GOPATH/src/k8s.io/sample-controller  # assuming your GOPATH has just one entry
```

Comment:
Is the comment about GOPATH having only one entry trying to explain the cd command? If so, the comment is unnecessary imo. Also, it might be confusing.

Reply:
> Is the comment about GOPATH having only one entry trying to explain the cd command?

Yes.

> Also, it might be confusing.

Ack, removed.